samples/Pan/ca.pan

```diff
@@ -0,0 +1,235 @@
+#
+# Generated by RepositoryTask on 11/10/14 12:21 PM
```

Comment:
This is a pretty big file with almost identical data repeated within the single `contents` dict: over 100 almost identical uncommented lines 😱!!  

Whilst this may be relevant to a real-world use, is not relevant to the structure of the language. Could you simplify this file or find a smaller real-world example?